modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/enricher/ContentParameterModelEnricher.java

```diff
@@ -85,8 +85,10 @@ private void configureDsl(ParameterDeclaration p) {
   }
 
   private void defaultToPayload(ParameterDeclaration p) {
-    p.setRequired(false);
-    p.setDefaultValue(PAYLOAD);
+    if (p.getDefaultValue() == null) {
+      p.setRequired(false);
```

Comment:
this is because if the user didn't give a default, then we force it as optional:defaultPayload... This was already like this before the change